packages/pyodide-kernel/src/worker.ts

```diff
@@ -449,48 +451,72 @@ export class PyodideRemoteKernel {
    * @param content The incoming message with the reply
    */
   async inputReply(content: any, parent: any) {
-    await this.setup(parent);
-
-    this._resolveInputReply(content);
+    // Should never be called as input_reply messages are returned via service worker
```

Comment:
We could explicitly throw an exception here in case this function is ever called in the future.